pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "PyHamcrest"
 description = "Hamcrest framework for matcher objects"
 readme = "README.rst"
 requires-python = ">= 3.8"
-license = { file = "LICENSE.txt" }
+license = "BSD-3-Clause"
```

Comment:
> I don't want to lose the license file

Hatchling does implement default globs if `license-files` is missing. So it would have been included anyway. It doesn't hurt to spell it out explicitly either though.
https://github.com/pypa/hatch/blob/hatchling-v1.26.3/backend/src/hatchling/metadata/core.py#L748-L749

Reply:
I don't guarantee that we'll always use Hatchling. Explicit, as always, is better than implicit :D